bin/setup

```diff
@@ -15,10 +15,13 @@ Dir.chdir APP_ROOT do
   puts "== Starting redis so database can be migrated =="
   system "redis-server &"
 
-  puts "\n== Preparing database =="
-  system "bin/rake db:setup db:test:prepare"
+  puts "\n== Preparing and seeding database =="
+  system "bin/rake db:setup"
   system "bin/rake seed_test_users_and_bikes"
 
+  puts "\n== Preparing test database (including parallelism) =="
+  system "bin/rake parallel:prepare"
```

Comment:
I think we'll want `bin/rake parallel:setup` for the first run. It can be time-consuming, though -- should we set up for parallelism eagerly?

> Create additional database(s)
> `rake parallel:create`
> Copy development schema (repeat after migrations)
> `rake parallel:prepare`
> Setup environment from scratch (create db and loads schema, useful for CI)
> `rake parallel:setup`
https://github.com/grosser/parallel_tests

Reply:
I haven’t successfully run `parallel:setup` - it errors. Is it still doing things that need to happen despite failing to finish?

Reply:
Sorry, typo — I meant `parallel:create`. I think `parallel:prepare` will fail if the additional test dbs don't exist yet. I'll test it though

Reply:
Disregard my last -- just repeated setup and it works fine 👍